src/QueryRenderer.ts

```diff
@@ -482,8 +483,11 @@ class QueryRenderChild extends MarkdownRenderChild {
         timeUnit: unitOfTime.DurationConstructor = 'days',
         amount = 1,
     ) {
-        const errorMessage = '⚠️ Postponement requires a date: due or scheduled.';
-        if (!task.dueDate && !task.scheduledDate) return new Notice(errorMessage, 10000);
+        const errorMessage = '⚠️ Postponement requires a happens date: start, due or scheduled.';
```

Comment:
A small and cosmetic thing, but before the dates were listed in priority order - `due or scheduled` - so I had though that to maintain this, the new message might be `due, scheduled or start`.

I'm not sure the new order `start, due or scheduled` is so logical.

Reply:
Fixed.

Reply:
What value does the `happens` do in the error message? To me, it makes it harder to read - happens is specific to searching, and isn't in the [date docs](https://publish.obsidian.md/tasks/Getting+Started/Dates).

Reply:
In this code, I see `happens` as being an implementation detail, rather than something that is of use to users.

Reply:
Fixed